app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -229,6 +229,9 @@ key --skip
 <%= @host.diskLayout %>
 <% end -%>
 
+<# TODO conditionals whether rhsm kickstart is supported #>
```

Comment:
I think it was RHEL 8.2 or 8.3 that added the rhsm/syspurpose directives.

Reply:
> Satellite support is only [available in RHEL 9.0](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.0_release_notes/index#enhancement_installer-and-image-creation) and later.

https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#rhsm says 

```
--server-hostname <subscription_service_server_url>

    Red Hat subscription service server URL.

    New in version RedHatEnterpriseLinux8.
```

Is that a docs bug?

Reply:
Yeah, looks like it - in the upstream kickstart docs - the `--server-hostname` option itself is not new, but since 9.0 its used exclusively for Satellite support (before it was basically used just for debugging).

I'll try to fix it. :)

Reply:
I've opened a PR on PyKickstart to fix the docs: https://github.com/pykickstart/pykickstart/pull/425